trade-finance-manager-api/src/v1/helpers/create-tasks-amendment.helper.api-test.js

```diff
@@ -0,0 +1,52 @@
+const { createAmendmentTasks, createTasksAutomaticAmendment, createTasksManualAmendment } = require('./create-tasks-amendment.helper');
+
+describe('createAmendmentTasks()', () => {
+  const mockFacility = {
+    requireUkefApproval: false,
+    tfm: {},
+  };
+
+  it('should create automatic amendment tasks if !requireUkefApproval and !nonDelegatedBank', () => {
+    const response = createAmendmentTasks(mockFacility.requireUkefApproval, mockFacility.tfm);
+
+    const expected = createTasksAutomaticAmendment();
+
+    expect(response).toEqual(expected);
+  });
+
+  it('should create manual amendment tasks if requireUkefApproval and !nonDelegatedBank', () => {
```

Comment:
Can we add a case which covers following please:
* When `not` a task should be created for a `NDB.
* When a manual deal tasks should be created for `NDB`.